src/test/java/edu/harvard/iq/dataverse/api/S3AccessIT.java

```diff
@@ -0,0 +1,71 @@
+package edu.harvard.iq.dataverse.api;
+
+import com.jayway.restassured.RestAssured;
+import static com.jayway.restassured.RestAssured.given;
+import com.jayway.restassured.path.json.JsonPath;
+import com.jayway.restassured.response.Response;
+import java.util.logging.Logger;
+import static javax.ws.rs.core.Response.Status.OK;
+import static junit.framework.Assert.assertEquals;
+import static org.hamcrest.CoreMatchers.equalTo;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.Matchers.startsWith;
+import org.junit.After;
+import org.junit.Assert;
+
+/**
+ *  NOTE: This test WILL NOT pass if your installation is not configured for Amazon S3 storage.
+ *  For S3 storage, you must set two jvm options: storage-driver-id and s3-bucket-name
+ *  Refer to the guides or to https://github.com/IQSS/dataverse/issues/3921#issuecomment-319973245
+ * @author bsilverstein
+ */
+public class S3AccessIT {
+    
+    private static final Logger logger = Logger.getLogger(S3AccessIT.class.getCanonicalName());
+
+    @BeforeClass
+    public static void setUp() {
+        RestAssured.baseURI = UtilIT.getRestAssuredBaseUri();
+        
+    }
+    
+    @Test
+    public void testAddDataFileS3Prefix() {
+        //create user who will make a dataverse/dataset
+        Response createUser = UtilIT.createRandomUser();
+        String username = UtilIT.getUsernameFromResponse(createUser);
+        String apiToken = UtilIT.getApiTokenFromResponse(createUser);
+        
+        Response createDataverseResponse = UtilIT.createRandomDataverse(apiToken);
+        String dataverseAlias = UtilIT.getAliasFromResponse(createDataverseResponse);
+        
+        Response createDatasetResponse = UtilIT.createRandomDatasetViaNativeApi(dataverseAlias, apiToken);
+        Integer datasetId = JsonPath.from(createDatasetResponse.body().asString()).getInt("data.id");
+        createDatasetResponse.prettyPrint();
+        
+        //upload a tabular file via native, check storage id prefix for s3
+        String pathToFile = "scripts/search/data/tabular/1char";
+        Response addFileResponse = UtilIT.uploadFileViaNative(datasetId.toString(), pathToFile, apiToken);
+        addFileResponse.prettyPrint();
+        addFileResponse.then().assertThat()
+                .body("data.files[0].dataFile.storageIdentifier", startsWith("s3://"));
```

Comment:
We may gain from having this as a constant. Maybe not worth creating a new file for just "s3://" & "swift://"